lib/rules/func-call-spacing.js

```diff
@@ -161,7 +167,20 @@ module.exports = {
                     },
                     messageId: "unexpectedNewline",
                     fix(fixer) {
-                        return fixer.replaceTextRange([leftToken.range[1], rightToken.range[0]], " ");
+                        const range = [leftToken.range[1], rightToken.range[0]];
```

Comment:
I believe it's a bug that this scenario still has autofix, and that it should have been removed while fixing #7787 issue (it hides bugs in the code reported by `no-unexpected-multiline`), so I'm not sure should we rather just remove this `fix()` now instead of upgrading it for optional calls.

Reply:
Good catch. I removed autofix that remove line breaks of non-optional calls to avoid hiding `no-unexpected-multiline` errors. I remained about optional calls as-is because `no-unexpected-multiline` doesn't report in that case.

Reply:
:+1: 

There's another alternative for handling comments, but we could discuss that later in #13411